tools/data-handler/src/utils/user-preferences.ts

```diff
@@ -57,7 +113,7 @@ export class UserPreferences {
     }
   }
 
-  public getPreferences() {
+  public getPreferences(): UserPreferencesObject {
```

Comment:
Would it make sense to remove the constructor and have this method have one parameter and make it static?

So instead of

```
const prefs = new UserPreferences(path).getPreferences(); 
```

You'd have
```
const prefs = UserPreferences.getPreferences(path);
```

Reply:
Is there a reason to prefer static classes over functions?